### PR TITLE
Add OBB's  test of corner case, add quaternion around_axis.

### PR DESCRIPTION
**Pull request by waizui**

### CODE REVIEW

File: del-geo-core/src/quaternion.rs [hunk 1]
```diff
@@ -119,3 +121,15 @@ pub fn from_axisangle(a: &[f32; 3]) -> [f32; 4] {
         (lena * half).cos(),
     ]
 }
+
+pub fn identity() -> [f64; 4] {
```
Comment (asukaminato0721) [inline del-geo-core/src/quaternion.rs:125]: try

```rs
pub const fn identity() -> [f64; 4] {
```

Comment (waizui) [reply del-geo-core/src/quaternion.rs:125]: cool, I think I will consider use const or inline by the time of doing optimization.

**Review by nobuyuki83 — COMMENTED** [review]
Looks good!
